Problem28 - Please review

```diff
@@ -0,0 +1,24 @@
+// Time Complexity :O(n)
```

Comment:
Good. Whats the brute force solution

Reply:
Brute force : Iterate through each element and check if there is any other value equal to( target - element)